src/main/scala/org/dberg/hubot/listeners/TestListener3.scala

```diff
@@ -0,0 +1,21 @@
+package org.dberg.hubot.listeners
+
+import org.dberg.hubot.Hubot
+import org.dberg.hubot.listeners.Listener.CallbackSuccess
+import org.dberg.hubot.models._
+
+class TestListener3(hubot: Hubot) extends Listener(hubot) {
+
+  val callback: Callback = {
```

Comment:
Fair enough lets leave it.  Can you bump to 0.3.1 and just change Direct() and Group() to DirectMessage() and GroupMessage() and we are good !